src/components/UI/Layout/Navigation/SideMenus/SideMenus.tsx

```diff
@@ -1,33 +1,101 @@
 import React, { useState } from 'react';
 import { NavLink } from 'react-router-dom';
 import { ListItem, ListItemIcon, ListItemText, List } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import clsx from 'clsx';
+import styles from './SideMenus.module.css';
 
 import { sideDrawerMenus } from '../../../../../config/menu';
 import ListIcon from '../../../ListIcon/ListIcon';
 
-export interface SideMenusProps {}
+export interface SideMenusProps {
+  opened: boolean;
+}
 
-const SideMenus: React.SFC<SideMenusProps> = () => {
+const useStyles = makeStyles({
```

Comment:
Wasn't successfully able to generalize this into a CSS file, so I would love to get some help with this (especially the selected + hover parts)!